snapshots/aviation_safety_network/2023-04-18/aviation_statistics.py

```diff
@@ -0,0 +1,95 @@
+"""Script to create a snapshot of dataset 'Aviation statistics'.
+
+In total, 3 snapshots will be created:
+* Data from their shared Google spreadsheet (from one specific sheet).
+* Data extracted directly from their page on aviation statistics by period.
+* Data extracted directly from their page on aviation statistics by nature.
```

Comment:
You could add a note explaining that the data for these two is sourced dynamically (and not from snapshot dvc metadata) from source HTML site. As a result, `source_data_url` in the DVC files is empty.